build/arm.sh

```diff
@@ -93,16 +103,31 @@ EOF
 
 mkdir -p ${STAGEDIR}/boot/msdos
 mount_msdosfs /dev/${DEV}s1 ${STAGEDIR}/boot/msdos
-
 mkdir -p ${STAGEDIR}/boot/msdos/overlays
-cp -p ${STAGEDIR}/boot/ubldr ${STAGEDIR}/boot/msdos/ubldr
-cp -p ${STAGEDIR}/boot/ubldr.bin ${STAGEDIR}/boot/msdos/ubldr.bin
+
+case "${PRODUCT_ARCH}" in
+armv6)
+	cp -p ${STAGEDIR}/boot/ubldr ${STAGEDIR}/boot/msdos/ubldr
+	cp -p ${STAGEDIR}/boot/ubldr.bin ${STAGEDIR}/boot/msdos/ubldr.bin
+	;;
+aarch64)
+	cp -p ${STAGEDIR}/boot/loader.efi ${STAGEDIR}/boot/msdos/bootaa64.efi
```

Comment:
@fichtner will you add this part of EFI loader into current arm.sh? taken from https://gitlab.com/HardenedBSD/hardenedBSD/blob/hardened/current/master/release/arm64/PINE64.conf#L37. This PR need to be closed because got a big conflict on arm.sh. Maybe I'll revert changes on everything except for nanopi device config for it to be merged.

Reply:
let's please close this and start on separate pieces again. so far RPI3 is not officially supported on 11.2 which makes me doubt we should try it now and I want the arm code to be more structured in their device files (u-boot code in there is better for readability).